curriculum/challenges/english/25-front-end-development/lecture-html-fundamentals/670803abcb3e980233da4768.md

```diff
@@ -0,0 +1,121 @@
+---
+id: 670803abcb3e980233da4768
+title: What Are Divs and Spans, and When Should You Use Them?
+challengeType: 11
+videoId: nVAaxZ34khk
+dashedName: what-are-divs-and-spans
+---
+
+# --description--
+
+Watch the video and answer the questions below.
```

Comment:
we are providing a transcript for all of these videos.
The transcripts will be pulled directly from the video itself since those recording might alter scripts once they start recording.

I think for now we should replace the answer the questions below with lorem ipsum since this will be eventually replace with the actual transcript

Reply:
I think the transcript will probably fit better somewhere else. Like below the videos. So it would be something like:
```
Watch the video and answer the questions below.
[video]
[transcript]
[questions]
```

So in that case, we would be fine to leave this?

Reply:
Yeah, that flow you laid out makes sense. So we can just leave this as is